data-processing/entities/sentences/extractor.py

```diff
@@ -82,6 +166,9 @@ def parse(self, tex_path: str, tex: str) -> Iterator[Sentence]:
         segmenter = pysbd.Segmenter(language="en", clean=False)
         # Record the current length of the plain text so account for the extractor bug
         length_so_far_in_plain_text = 0
+
+
+        current_section, current_table, current_figure = False, False, False
```

Comment:
nit pick: if current_section is a string, should its default be None? And if the other two variables track whether the sentence is in a table or figure but not the identity of the current table or figure, are these better names?

```suggestion
        current_section, in_table, in_figure = None, False, False
```

Reply:
Yes, I changed the names from `in_{table,figure}` to `is_sentence_in_{table,figure}`. Also, type for `current_section` is changed to `str`.